logging/unit_tests/test_client.py

```diff
@@ -562,7 +563,8 @@ def test_get_default_handler_app_engine(self):
                                 credentials=_make_credentials(),
                                 use_gax=False)
 
-        with _Monkey(_MUT, _LOG_PATH_TEMPLATE='{pid}'):
+        temp_log_path = os.path.join(tempfile.mkdtemp(), '{pid}')
```

Comment:
Yeah I saw that but I what I didn't realize was...
>The directory is readable, writable, and searchable only by the creating user ID.

@waprin thanks for pointing it out in your notes and thanks @tseaver for the fix!